src/screens/AuthScreen.tsx

```diff
@@ -58,6 +58,7 @@ export const AuthScreen: React.FC<AuthScreenProps> = () => {
         mode="contained"
         icon={({ size }) => (
           <Image
+            // eslint-disable-next-line @typescript-eslint/no-require-imports
```

Comment:
I'm seeing many such comments in code, maybe we should consider moving this rule to the eslint config file instead then

Reply:
Potentially - normally I would say if it's only a few situations, and the rule apples to most of the codebase, keep it in - but since it seems to be an intentional part of the Expo asset system, maybe we should remove it